lms/services/grouping.py

```diff
@@ -8,6 +9,13 @@
 from lms.services.upsert import bulk_upsert
 
 
+class GroupingInfo(TypedDict):
```

Comment:
If you are  thinking what's up with this guy and "TypeDict"...


It could be dataclass but then it will need to be imported and we'll go from dict -> dataclass -> dict. With this we only go from dict with one set of keys to another dict.

In any case is for documentation mostly, pyright also complains if I pass the wrong key (or not all of them).